install/kubernetes/helm/istio/values.yaml

```diff
@@ -135,6 +135,15 @@ global:
     enableHttps: false
 
   proxy:
+    # Configuration for the proxy init container
+    init:
+      resources:
+        limits:
+          cpu: 10m
+          memory: 10Mi
```

Comment:
can you keep the limits the same? 10mi is really low as a default

Reply:
Ah yes, that was unintentional.